airbyte-integrations/connectors/source-marketstack/manifest.yaml

```diff
@@ -0,0 +1,505 @@
+version: 5.15.0
+
+type: DeclarativeSource
+
+description: >-
+  Marketstack provides data from 72 global stock exchanges.
+
+  Using this connector we can extract Historical Data , Splits and Dividends
+  data !
+
+check:
+  type: CheckStream
+  stream_names:
+    - Exchanges
+
+definitions:
+  streams:
+    Exchanges:
+      type: DeclarativeStream
+      name: Exchanges
+      primary_key:
+        - mic
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: exchanges
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - data
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: offset
+          page_size_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: limit
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 1000
+            inject_on_first_request: true
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/Exchanges"
+    Tickers:
+      type: DeclarativeStream
+      name: Tickers
+      primary_key:
+        - symbol
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /exchanges/{{ stream_partition.mic }}/tickers
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - data
+              - tickers
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: offset
+          page_size_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: limit
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 1000
+            inject_on_first_request: true
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: mic
+              partition_field: mic
+              stream:
+                $ref: "#/definitions/streams/Exchanges"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/Tickers"
+    Historical Data:
```

Comment:
This stream must be incremental. Please fix that.

Reply:
done!!